stores/amazon.py

```diff
@@ -162,7 +169,11 @@ def __init__(self, notification_handler, headless=False):
     def is_logged_in(self):
         try:
             text = wait_for_element(self.driver, "nav-link-accountList").text
-            return "Hello, Sign in" not in text
+            for sign_in in SIGN_IN_TEXT:
+                if sign_in in text:
+                    return False
+
+            return True
```

Comment:
Can we make this ```return text not in SIGN_IN_TEXT```

Reply:
Thanks for the feedback, not sure if that would work since the sentences in `SIGN_IN_TEXT` are only part of the `text` var, so it's not an exact match.

For example, for .co.uk var text also includes the Accounts & Lists part, so either we can edit those SIGN_IN_TEXT sentences to include the full string, or check it like it is currently.

Reply:
I think we can do something like:
```
return not any(sign_in in text for sign_in in SIGN_IN_TEXT)
```

Reply:
Ah, thanks, yes that works fine!